src/services/expensesService.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+let expenses = [];
```

Comment:
You can use const in this place. 
I see you manipulate with an array, easy way to use const and avoid reassigned.